server/ERNI.PBA.Server.DataAccess/Repository/IInvoiceImageRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using ERNI.PBA.Server.DataAccess.Model;
+
+namespace ERNI.PBA.Server.DataAccess.Repository
+{
+    public interface IInvoiceImageRepository
+    {
+        Task<IdNamePair[]> GetInvoiceImagesIdNamePairs(int requestId, CancellationToken cancellationToken);
```

Comment:
Why IdNamePair? Call it InvoiceImageProjection, the method can be GetInvoiceImages